govuk-lint.gemspec

```diff
@@ -23,5 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.3"
 
   spec.add_dependency "rubocop", "~> 0.49.0"
+  spec.add_dependency "rubocop-rspec"
```

Comment:
Should we pin to a specific version, like we have with rubocop; so that we can manage when we introduce new cops and default settings etc...?

Reply:
Personally I think it's an unnecessary restriction and acts to slow down our adoption of the latest improvements. However, I'm happy to add the constraint if that's what people prefer and/or for consistency.

Reply:
Without it doesn't it mean that you and I can both install govuk-lint at the same version, but get different versions of rubocop-rspec and thus drastically different results from linting our rspec files?

Reply:
I guess so - I'm just not that convinced the differences would be important. However, I've added a fixup commit to add the constraint as requested.

Reply:
0a3c043bddc718bb30c1dba63cc390c5dd707beb

Reply:
Maybe I'm just being cautious as I've seen plenty projects get told that code is to look one way with rubocop vX and then to change it's mind and ask it to look different in rubocop vX+1, or for the cop names to change between versions and your default settings stop applying.  Some of this is mitigated by the autocorrectable cops, but not everything is.

I don't think new releases of rubocop shouldn't just be installed without careful review of the new rules and their config to make sure you're not just introducing style changes you don't actually want.  Maybe rubocop-rspec isn't as changeable as rubocop itself though, but I don't think it hurts to make explicit the version we've decided we're happy to be using.

Anyway, thanks for acquiescing.